projects/knora-ui/src/lib/viewer/pipes/knoradate.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { KnoraDate } from '@knora/api';
+
+@Pipe({
+  name: 'knoraDate'
+})
+export class KnoraDatePipe implements PipeTransform {
+
+  // TODO: use the optional format parameter and return a string with that format
+  transform(value: KnoraDate, format?: string): string {
+    let formattedDate: Date = new Date();
```

Comment:
I think you should explicitly check if it is a `KnoraDate` since a `FormControl`'s value could also be `null` or it could simply be used wrong:

```ts
if (!(value instanceof KnoraDate)) {
     return '';
}
```

Reply:
fixed in 37865b7
now gives a console error and returns an empty string. Should the user be provided with some text?

Reply:
My rational was to prevent a type error. I think an empty string is fine.